tabular/src/autogluon/tabular/predictor/predictor.py

```diff
@@ -299,11 +300,13 @@ def fit(self,
         hyperparameters : str or dict, default = 'default'
             Determines the hyperparameters used by the models.
             If `str` is passed, will use a preset hyperparameter configuration.
-                Valid `str` options: ['default', 'light', 'very_light', 'toy']
+                Valid `str` options: ['default', 'light', 'very_light', 'toy', 'multimodal']
                     'default': Default AutoGluon hyperparameters intended to maximize accuracy without significant regard to inference time or disk usage.
                     'light': Results in smaller models. Generally will make inference speed much faster and disk usage much lower, but with worse accuracy.
                     'very_light': Results in much smaller models. Behaves similarly to 'light', but in many cases with over 10x less disk usage and a further reduction in accuracy.
                     'toy': Results in extremely small models. Only use this when prototyping, as the model quality will be severely reduced.
+                    'multimodal': [EXPERIMENTAL] Trains a multimodal transformer model alongside tabular models. Requires text columns, a GPU, and CUDA-enabled MXNet.
+                        When combined with 'best_quality' `presets` option, this can achieve extremely strong results in multimodal datasets.
```

Comment:
results in multimodal data tables that contain columns with text in additional to numeric/categorical columns.

Reply:
Added